spec/models/intake_spec.rb

```diff
@@ -1041,4 +1041,27 @@
       expect(intake.relevant_intake_document_types).to eq doc_types
     end
   end
+
+  describe "#itin_applicant?" do
```

Comment:
Not a blocker for this pull request:

At some point in the near future (1-5 days), since [we are going to have to let Hub users edit the ITIN-applicant nature of a client](https://cfa.slack.com/archives/GT1B45BS5/p1644533164143279), I think we should change `itin_applicant?` to be a boolean property on `Intake`. That way, Hub users would be editing Intake data rather than Triage data. In that case, we'd need to copy the triage answer onto the Intake, similar to how the backtaxes years work.

Would you agree with that?

This is not a request for a change on this pull request.

Reply:
the intake is not created at the time at which someone answers the triage question, so theres a logic change that will be needed to make that happen. I don't disagree with you, but that feels out of scope for this ticket.

Reply:
Agreed, out of scope for this ticket, just offering a heads-up to you and looking to validate the idea. Sounds like you would consider such a change peaceful to do in the future.